Stripe/STPBancontactSourceInfoDataSource.m

```diff
@@ -27,7 +27,7 @@ - (instancetype)initWithSourceParams:(STPSourceParams *)sourceParams {
     return self;
 }
 
-- (STPSourceParams *)completedSourceParams {
+- (STPSourceParams *)completeSourceParams {
```

Comment:
Hm, so I think of these methods as being pretty tied to the VC – they construct STPSourceParams from the data source's contents, and then perform some very light validation that just checks whether the fields in the form have been filled.

I'm not sure there's a need for a more in-depth STPSourceParams validator at this point – STPSourceInfoVC will do its best to ensure the returned SourceParams object is filled, and I think we can account for edge cases like not setting your returnURL by printing an error when you initialize the view controller.